integration_tests/models/any_type_column_anomalies_no_timestamp.sql

```diff
@@ -0,0 +1,37 @@
+-- depends_on: {{ ref('any_type_column_anomalies_validation') }}
```

Comment:
cosmetics - I'd call this model - no_timestamp_table_anomalies
Also - currently you only validate table_anomalies and there is no coverage for column anomalies so maybe worth adding a new fake data instead of relying on  any_type_column_anomalies_training.
I'd create a new fake training file that contains X rows and the validation file will contain X * 2. In terms of the columns you can have one column per column type (number, string, any type) and simulate one anomaly per type so we will have sanity also when there is no timestamp. WDYT?